includes/AppLeave.php

```diff
@@ -75,6 +75,27 @@
 	require_once "../LocalSettings.php";
 	require_once "Globals.php";
 	
+	// validate dates
+	//
+	$dtnow = date('Y-m-d');
+	if( isset( $_POST['Start_Date'] ) )
+	{
+		if( strtotime($_POST['Start_Date']) < strtotime($dtnow) )
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 1 space after IF keyword; 0 found
* Expected 0 spaces after opening bracket; 1 found
* Expected 0 spaces before closing bracket; 1 found
* Expected 1 space after closing parenthesis; found 9

<!-- Comment by [SideCI](https://sideci.com) -->